observatory_platform/dataset_api.py

```diff
@@ -116,14 +117,14 @@ def from_dict(_dict: Dict) -> DatasetRelease:
             dag_id=_dict["dag_id"],
             entity_id=_dict["entity_id"],
             dag_run_id=_dict["dag_run_id"],
-            created=bq_timestamp_to_pendulum(_dict["created"]),
-            modified=bq_timestamp_to_pendulum(_dict["modified"]),
-            data_interval_start=bq_timestamp_to_pendulum(_dict.get("data_interval_start")),
-            data_interval_end=bq_timestamp_to_pendulum(_dict.get("data_interval_end")),
-            snapshot_date=bq_timestamp_to_pendulum(_dict.get("snapshot_date")),
-            partition_date=bq_timestamp_to_pendulum(_dict.get("partition_date")),
-            changefile_start_date=bq_timestamp_to_pendulum(_dict.get("changefile_start_date")),
-            changefile_end_date=bq_timestamp_to_pendulum(_dict.get("changefile_end_date")),
+            created=_dt(_dict["created"]),
+            modified=_dt(_dict["modified"]),
+            data_interval_start=_dt(_dict.get("data_interval_start")),
+            data_interval_end=_dt(_dict.get("data_interval_end")),
+            snapshot_date=_dt(_dict.get("snapshot_date")),
+            partition_date=_dt(_dict.get("partition_date")),
+            changefile_start_date=_dt(_dict.get("changefile_start_date")),
+            changefile_end_date=_dt(_dict.get("changefile_end_date")),
```

Comment:
These changes should be reverted, as the BigQuery string timestamps need to be converted to pendulum DateTime objects inside from_dict.